ecl/resource_ecl_network_static_route_v2.go

```diff
@@ -237,12 +243,11 @@ func resourceNetworkStaticRouteV2Delete(d *schema.ResourceData, meta interface{}
 	config := meta.(*Config)
 	networkClient, err := config.networkV2Client(GetRegion(d, config))
 	if err != nil {
-		return fmt.Errorf("Error creating ECL network client: %s", err)
+		return fmt.Errorf("error creating ECL network client: %w", err)
 	}
 
-	err = static_routes.Delete(networkClient, d.Id()).ExtractErr()
-	if err != nil {
-		return fmt.Errorf("Errof deleteting ECL Static route: %s", err)
+	if err = static_routes.Delete(networkClient, d.Id()).ExtractErr(); err != nil {
+		return fmt.Errorf("error deleting ECL Static route: %w", err)
```

Comment:
すでにリソースが存在しない場合はStateを削除するだけで良いので以下のように修正してください。
```suggestion
		return CheckDeleted(d, err, "error deleting ECL Static route")
```